bindata/etcd/cluster-restore.sh

```diff
@@ -96,8 +96,14 @@ if [ -d "${ETCD_DATA_DIR}/member" ]; then
   fi
   echo "Moving etcd data-dir ${ETCD_DATA_DIR}/member to ${ETCD_DATA_DIR_BACKUP}"
   mv ${ETCD_DATA_DIR}/member ${ETCD_DATA_DIR_BACKUP}/
+  # Remove any other previously archived member directories
+  rm -rf ${ETCD_DATA_DIR}/*
```

Comment:
This is needed, when one time I was restoring on a different master node, and found that there is archived member data in the directory /var/lib/etcd/restore-data-stamp-member-data. That disallows recovery pod to succeeded.